netlify.toml

```diff
@@ -3,8 +3,8 @@
   functions = "./functions"
   publish = "public/"
 [context.staging.environment]
-  ROUTER_REGISTRY_URL_ENV_VAR = "https://struggle-slack-app-beta.netlify.com/slack-app-registry.json"
-  GO_IMPORT_PATH = "github.com/junior-dev-struggle-bus/struggle-slack-app"
+  ROUTER_REGISTRY_URL_ENV_VAR = "https://lisas-first-serverless-func-test.netlify.com/slack-app-registry.json"
+  GO_IMPORT_PATH = "github.com/kimlisa/struggle-slack-app"
```

Comment:
Mark can correct me if I'm wrong, but these should probably be reverted back to the struggle-slack-app-beta URL and JDSB github before merging. :) I don't think it would hurt much right now, but the next time someone else tried to commit changes, they would be pointing to your registry and github.

Reply:
Yes, you'll want to revert this back to the way it was. In the future, you can create a branch called, "lisasdev", and add an entry in the netlify.toml like so:

[context.lisasdev.environment]
ROUTER_REGISTRY_URL_ENV_VAR = "..."
GO_IMPORT_PATH = "..."